### DISCUSSION TITLE
V6 historical alerts styling: the left border is gone

### DISCUSSION THREAD

**Discussion by masterjyq** [ccfos/nightingale#1419] [General]
V5 的历史告警, 有个样式`*-left-border`表示告警级别,  和告警恢复, 挺好用的 v6 好像没了, 看代码是这块Table的代码直接重构了. 可以考虑加回来, 或者用其他方式展示
![image](https://user-images.githubusercontent.com/29848365/224592882-0d0d9975-94b3-4114-9f22-3a34277c75f1.png)

**Comment by dreamlsh** [comment]
还没试过v6,可以只升级前端吗

**Reply by masterjyq** [reply]
不行, 整体架构变了,  以前集群的概念成了数据源, 开始向Grafana 靠近了

**Comment by masterjyq** [comment]
样式已经有了
